js/modal.js

```diff
@@ -0,0 +1,102 @@
+const projectList = {
+  firstProject: {
+    popupHeading: 'Multi-Post Stories 1',
+    languages: ['html', 'Bootstrap', 'Ruby on Rails'],
+    images: 'images/mb-modal.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    liveServerbtn: 'https://basnetlaxmi.github.io/my-portfolio/',
+    sourceCode: 'https://github.com/basnetlaxmi/my-portfolio ',
+
+  },
+  secondProject: {
+    popupHeading: 'Multi-Post Stories 2',
+    languages: ['html', 'Bootstrap', 'Ruby on Rails'],
+    images: 'images/mb-modal.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    liveServerbtn: 'https://basnetlaxmi.github.io/my-portfolio/',
+    sourceCode: 'https://github.com/basnetlaxmi/my-portfolio ',
+  },
+  thirdProject: {
```

Comment:
- [ ] This was a good job adding the project details in a JavaScript object, however, the container of all the project objects should be an array and not an object as mentioned in the requirement 👇 
 
![image](https://user-images.githubusercontent.com/55465133/131918080-e83684e8-3880-4f0d-ac5f-e15d8ce4545c.png)

Kindly let's have all the project objects in an array instead 😉